ee/fleetctl/updates.go

```diff
@@ -637,7 +637,7 @@ func newPassphraseHandler() *passphraseHandler {
 	return &passphraseHandler{cache: make(map[string][]byte)}
 }
 
-func (p *passphraseHandler) getPassphrase(role string, confirm bool) ([]byte, error) {
+func (p *passphraseHandler) getPassphrase(role string, confirm, change bool) ([]byte, error) {
```

Comment:
Looks like the added `change` argument is unused?

Reply:
Ah, nevermind, I see that the func is passed to the `tuf` package elsewhere, so that's probably required for the function signature it expects.

Reply:
I'll add a comment and file an issue.